doc/changes/latest.inc

```diff
@@ -76,6 +76,16 @@ Enhancements
 
 - Static type checkers like Pylance (comes with VS Code) now display the parameters of many more functions correctly, largely improving overall usability for VS Code users (:gh:`8862` by `Richard Höchenberger`_)
 
+- Add ``match_alias`` parameter to :meth:`mne.io.Raw.set_montage` and related functions to match unrecognized channel location names to known aliases (:gh`8799` **by new contributor** |Zhi Zhang|_)
+
+API changes
+~~~~~~~~~~~
+- None
+
+Deprecations
```

Comment:
Did we decide to make a new section called `Deprecations`? Usually this is what `API changes` is for I think -- ENH/BUG/API has been the standard section naming and order for all our releases. Let's not reorganize/rename/reorder in this PR, but if this important to you we can discuss and then redo all of the lists.

Reply:
Yes we only have API section

Reply:
> Did we decide to make a new section called `Deprecations`?

Nope, this is a new proposal here :)

> Let's not reorganize/rename/reorder in this PR, but if this important to you we can discuss and then redo all of the lists in a separate PR.

Will revert this here and make a new proposal in a separate PR